src/patternfly/components/NotificationDrawer/examples/NotificationDrawer.md

```diff
@@ -38,6 +41,9 @@ cssPrefix: pf-c-notification-drawer
     {{/notification-drawer-header-status}}
     {{#> notification-drawer-header-action}}
       {{#> dropdown id=(concat notification-drawer--id "-header-action") dropdown-menu--modifier="pf-m-align-right" dropdown--IsActionMenu="true" dropdown-toggle--modifier="pf-m-plain" dropdown--HasKebabIcon="true" aria-label="Actions"}}{{/dropdown}}
+      {{#> button button--modifier="pf-m-plain" button--attribute='aria-label="Close"'}}
```

Comment:
I wonder if we should have a BEM element for this button, so that in the future, if we want to space/style the close button differently than other buttons a user might put here, we have an element to target?

Reply:
do you think i should call it `.pf-c-notification-drawer__header-action-button` or `.pf-c-notification-drawer__header-action-close`

Reply:
"close" would be inline with what we use in other components.